qiskit_experiments/test/mock_iq_backend.py

```diff
@@ -172,15 +171,27 @@ class MockIQBackend(FakeOpenPulse2Q):
 
```

Comment:
Wrong class doc. Change to something like: """A mock backend for testing with IQ data."""

Reply:
Changed as suggested.